fighthealthinsurance/fax_views.py

```
@@ -39,8 +39,27 @@ def get(self, request, **kwargs):
+        try:
+            ctx = super().get_context_data(**kwargs)
+            # Get the form object because it's a form view.
+            form = ctx.get("form")
+
+            # Template expects `fax_form`, not `form`
+            ctx["fax_form"] = form
+            if self.request.method == "POST" and ctx:
+                ctx.setdefault("appeal", self.request.POST.get("completed_appeal_text", ""))
+                ctx.setdefault("denial_id", self.request.POST.get("denial_id"))
+                ctx.setdefault("user_email", self.request.POST.get("email"))
+            return ctx
```

Ruff (0.14.6)

Consider moving this statement to an `else` block

(TRY300)